### PR TITLE
Adding my name to ReadMe file

### PR DESCRIPTION
**Pull request by brandondombrowsky**
This was practice to ensure that it would take 2 PRs from the non owner (me)

### CODE REVIEW

**Comment by VictorHugoNava** [comment]
LGTM

**Review by VictorHugoNava — COMMENTED** [review]
LGTM

**Review by VictorHugoNava — APPROVED** [review]
LGTM

**Comment by brandondombrowsky** [comment]
I put a pin on this for a second earlier to test a few things. I wanted to see the dummy project code in my branch and an extra word in my own branch's ReadMe. I see that now. 

But I wasn't sure if merges were still allowing only one approval (I know I can still do it). So go ahead and approve this one and we can test if it holds up to the requirement of 2 approvals (not just mine). This was something Poul helped with, so it'd be cool if it worked. 

As for the PRs all approved for me, I got what I excected. So Yay: We have all our names in the Main ReadMe, and I have an extra line in the ReadMe of my branch (as a test).

Finally, don't forget to git pull the most recent version onto your own sustem.

File: README.md
Comment (alyoshenka) [inline README.md:10]: Nice for testing, but since we're doing an "official" review process: do we need this line? (I'll approve if you want it here)

**Review by alyoshenka — CHANGES_REQUESTED** [review]

File: README.md
Comment (brandondombrowsky) [reply README.md:10]: I know it seems tedious, but if you would, that would be great. Here are three things I am checking:
1. Will a change this small and seemingly innocuous merge automatically?
2. How will the merge control box change? Will we get the green manual merge button?
3.  Though I have added as many restrictions as I can to prevent me from becoming an evil genius on this project, I STILL have an override option: 
image
I want to see if 2 approvals makes that go away.

Comment (alyoshenka) [reply README.md:10]: roger dodger

**Review by alyoshenka — APPROVED** [review]

**Pull request closed without merging**
